rmgpy/data/kinetics/family.py

```diff
@@ -924,6 +924,15 @@ def save_training_reactions(self, reactions, reference=None, reference_type='',
 
         # Add new reactions to training depository
         for i, reaction in enumerate(reactions):
+            # Convert coverage dependence label to species label
+            if isinstance(entry.data, (SurfaceArrhenius, StickingCoefficient, SurfaceArrheniusBEP, StickingCoefficientBEP)):
+                if reaction.kinetics.coverage_dependence:
+                    coverage_dependence = {}
+                    reaction.kinetics.coverage_dependence = coverage_dependence
+                    for key, value in reaction.kinetics.coverage_dependence.items():
+                        coverage_dependence.update({species_dict[key]: value})
```

Comment:
isn't 
```suggestion
                        coverage_dependence[species_dict[key]] = value
```
simpler and quicker?